src/__snapshots__/App.test.jsx.snap

```diff
@@ -0,0 +1,43 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
@neelxie why is this file still tracked?!

Reply:
It has been added to the .gitignore file as you can see. I do recommend you take a moment and check out this resource on the topic https://jestjs.io/docs/en/snapshot-testing.html#should-snapshot-files-be-committed